pkg/services/accesscontrol/resourcepermissions/service.go

```diff
@@ -368,3 +369,38 @@ func (s *Service) declareFixedRoles() error {
 
 	return s.service.DeclareFixedRoles(readerRole, writerRole)
 }
+
+type ActionSetService interface {
+	// ActionResolver defines method for expanding permissions from permissions with action sets to fine-grained permissions.
+	// We use an ActionResolver interface to avoid circular dependencies
+	accesscontrol.ActionResolver
+
+	ResolveAction(action string) []string
```

Comment:
could we add comments to indicate the difference between resolve action `ResolveAction` and `ResolveActionSet`.

Both return a `[]string`